File: libs/common/models/vw-pmp-psispecialization-by-category-list.ts



export class VwPmpPsispecializationByCategoryList {
    public cipSubSeriesCode: string = '';
    public cipSubSeries: string = '';
    public providerName: string = '';
    public logoUrl: string = '';
    public programName: string = '';
    public specializationName: string = '';
    public specializationId: number = 0;
    public programNumber: number;
    public specializationNumber: number;
    public programCode: string = '';
    public specializationCode: string = '';
    public sessionStartDate: Date;
    public sessionEndDate: Date;

    constructor(init? : Partial<VwPmpPsispecializationByCategoryList>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwPmpPsispecializationByCategoryList.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwPmpPsispecializationByCategoryList
    {
        public string CipSubSeriesCode { get; set; }
        public string CipSubSeries { get; set; }
        public string ProviderName { get; set; }
        public string LogoUrl { get; set; }
        public string ProgramName { get; set; }
        public string SpecializationName { get; set; }
        public int SpecializationId { get; set; }
        public int? ProgramNumber { get; set; }
        public int? SpecializationNumber { get; set; }
        public string ProgramCode { get; set; }
        public string SpecializationCode { get; set; }
        public DateTime? SessionStartDate { get; set; }
        public DateTime? SessionEndDate { get; set; }
    }
}
